main.go

```diff
@@ -1,29 +1,50 @@
 package main
 
 import (
-	"fmt"
+	"os"
 
-	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-steputils/stepenv"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/env"
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
-	"github.com/bitrise-steplib/steps-activate-ssh-key/activatesshkey"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-steplib/steps-activate-ssh-key/sshkey"
+	"github.com/bitrise-steplib/steps-activate-ssh-key/step"
 )
 
+// TODO debug log config? os.GetEnv? late init?
+var logger = log.NewLogger(false)
```

Comment:
How about re-introducing the `SetEnableDebugLog()` method that would switch the logging mode at runtime? I think this would be the least painful option, it's a common pattern that we only know the log level after parsing the step input.

Reply:
resolved in https://github.com/bitrise-steplib/steps-activate-ssh-key/pull/25/commits/13a620e0d304b826ad605de91add46a46ec351d1